test/c3/appframework/appframework_suite_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2018-2021 Splunk Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package c3appfw
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	"github.com/splunk/splunk-operator/test/testenv"
+)
+
+const (
+	// PollInterval specifies the polling interval
+	PollInterval = 5 * time.Second
+
+	// ConsistentPollInterval is the interval to use to consistently check a state is stable
+	ConsistentPollInterval = 200 * time.Millisecond
+	ConsistentDuration     = 2000 * time.Millisecond
+)
+
+var (
+	testenvInstance  *testenv.TestEnv
+	testSuiteName    = "c3appfw-" + testenv.RandomDNSName(3)
+	appListV1        []string
+	appListV2        []string
+	testDataS3Bucket = os.Getenv("TEST_BUCKET")
+	testS3Bucket     = os.Getenv("TEST_INDEXES_S3_BUCKET")
+	s3AppDirV1       = "appframework/regressionappsv1/"
```

Comment:
Do we wanna add these as env variables?

Reply:
Kept them here for now because of upcoming git action migration.